### ISSUE TITLE
Compatibility with timescaledb

### ISSUE THREAD

**Issue by ewoolsey** [sraoss/pg_ivm#88]
Anyone know if there's any change this is compatible with hyper tables? Continuous aggregates don't seem to be flexible enough for my use case and I'm wondering if pg_ivm would work here.

**Comment by yjhjstz** [comment]
no, pg_ivm do not support PARTITIONED TABLE.

**Comment by yugo-n** [comment]
Yes, pg_ivm doesn't support partitioned table or table inheritance for now.
I would like to support partitioned table in future, but not yet implemented.

**Comment by srivathsanvlb** [comment]
Please try to implement. I need it for a Hypertable running in Azure flexible server.
